libs/couch2pg/migrations/001.do.2318-standardise-schema.sql

```diff
@@ -0,0 +1,19 @@
+-- get everyone to baseline what existed before we added migrations
+CREATE TABLE IF NOT EXISTS couchdb (doc jsonb);
+
+-- IF NOT EXISTS doesn't exist (!) in Postgres 9.4, so do this silliness instead
+DO $$
+BEGIN
+
+IF NOT EXISTS (SELECT 1 FROM pg_class c WHERE c.relname = 'couchdb_doc_uuid') THEN
```

Comment:
Or we could move to PG9.5 :-)

I agree though, if we start doing this a lot we can add it as a permanent function into a migration script and use it elsewhere.